speech-api/SpeechSynthesis-speak-ownership.html

```diff
@@ -1,10 +1,14 @@
 <!doctype html>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
 <iframe></iframe>
 <script>
 // using an utterance for different SpeechSynthesis instances should throw
-test(t => {
+var t = async_test(
+  'Using the same SpeechSynthesisUtterance with two SpeechSynthesis instances');
+test_driver.click(document.body).then(t.step_func(() => {
```

Comment:
If you use `test_driver.bless`, this test will be easier to run manually. (Caveat: I haven't tried it myself.)

Reply:
Done.